test/coinselector.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { Coin } from '../src/wallet/coin';
+import { CoinSelector } from '../src/wallet/coin-selector.ts';
+import { Transaction } from 'bitcoinjs-lib';
+
+describe('CoinSelector', () => {
+    let coinSelector: CoinSelector;
+    const feeRate: number = 5;
```

Comment:
nit: typescript will infer the type so `const feeRate = 5;`

Reply:
Okay.